compiler-core/src/type_.rs

```diff
@@ -1920,6 +1920,25 @@ pub fn register_import(
             }
 
             if unqualified.is_empty() {
+                let imported_name = module
```

Comment:
Do we not need to do this when `unqualified` is not empty?

Reply:
Yeah, we need it. Should be fixed.